recipes/openmeeg/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "2.4.0.dev" %}
+
+package:
+  name: openmeeg
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmeeg/openmeeg/archive/2.4-rc2.tar.gz
+  sha256: 254eb82717ffec8b79618c281fe5c87734b2f8d804ef369d414d91b1b71cd163
+
+build:
+  number: 0
+  skip: true  # [win32]
+  msvc_compiler: 14.0
+  features:
+    - vc14  # [win]
+
+requirements:
+  build:
+    - cmake >=3.8
+    - toolchain
+    - python
+    - numpy
+    - swig
+    - hdf5 1.8.18|1.8.18.*
```

Comment:
Does this need `hdf5` version `1.8.18`? Currently we are moving from `1.10.1` to `1.10.2`.

ref: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/1f9b4021bfaa830eef6786ac373b9b10478aa2da/recipe/conda_build_config.yaml#L268-L269

Reply:
This is a libmatio requirement.

Reply:
https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.25640#L426

Reply:
maybe I should open an issue to libmatio.

Reply:
Sorry for missing this.

Yes, opening an issue (or better yet a PR) would be good. Is there one already open?